src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2188,11 +2188,12 @@ def create_deploy_webapp(cmd, name, location=None, sku=None, dryrun=False):  # p
 
     # determine the details for app to be created from src contents
     lang_details = get_lang_from_content(src_dir)
+    print(lang_details)
```

Comment:
get rid of this?

Reply:
oops yes

Reply:
How does `get_lang_from_content` work if there is no project file? Does it determine project type based on file extension?